packages/fluentui/react-northstar/src/components/Tree/Tree.tsx

```diff
@@ -196,7 +196,9 @@ export const Tree: ComponentWithAs<'div', TreeProps> &
   const [activeItemIds, setActiveItemIdsState] = useAutoControlled<string[]>({
     defaultValue: props.defaultActiveItemIds,
     value: props.activeItemIds,
-    initialValue: expandedItemsGenerator(items),
+    initialValue: props.activeItemIds
+      ? [...expandedItemsGenerator(items), ...props.activeItemIds]
+      : expandedItemsGenerator(items),
```

Comment:
I don't think that this change is valid as it affects only `initialValue`. What will happen if `defaultActiveItemIds` are defined?

Reply:
This one is tricky, if we remove the changes it will ignore the initial values passed to `activeItemIds` in the next item click/expand

Reply:
Removed